test/helpers/fakeGrpcTransport.ts

```diff
@@ -11,6 +11,14 @@ function frameResponse(request: Message): Uint8Array {
   return new Uint8Array(frame);
 }
 
+export function frameRequest(request: Message): ArrayBufferView {
```

Comment:
[nit] Could you please add a comment on why this is needed? This will allow external contributors to get up to speed with the code base easier.

Reply:
Oops sorry, missed this, will backfill :) <3